### PR TITLE
Suggestions

### PR DESCRIPTION
**Pull request by matiaslopezd**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -29,28 +29,21 @@ export default {
   name: 'app',
   components: { Header, UsersDashboard, StartupChart },
   async mounted() {
+    this.loadTheme();
```
Comment (matiaslopezd) [inline src/App.vue:32]: This improves the UX, avoiding the dark theme will load after data

File: src/App.vue [hunk 2]
```diff
@@ -29,28 +29,21 @@ export default {
...
     },
     toggleTheme() {
-      const { classList } = document.documentElement
-      if (classList.contains('dark')) localStorage.theme = ''
-      else localStorage.theme = 'dark'
-      classList.toggle('dark')
+      const isDarkActive = document.documentElement.classList.contains('dark');
+      const action = (isDarkActive) ? 'removeItem' : 'setItem';
+      localStorage[action]('theme', 'dark');
+      document.documentElement.classList.toggle('dark')
```
Comment (matiaslopezd) [inline src/App.vue:46]: Very small suggestion around how toggle dark theme :smile:

File: src/services/UsersService.js [hunk 3]
```diff
@@ -2,9 +2,9 @@ export const getUsers = async () => {
   const userResponse = await fetch('/api/users')
   const { users } = await userResponse.json()
 
-  users.forEach(({ sessions }, idx) => {
+  users.forEach(({ sessions }, index) => {
```
Comment (matiaslopezd) [inline src/services/UsersService.js:5]: Ever use complete sentences! Some inexperienced developers could be confused.

Comment (matiaslopezd) [reply src/services/UsersService.js:5]: And obviously is easy to read with a lot of code.

**Review by matiaslopezd — COMMENTED** [review]
These are the suggestions we have for you! Congrats!

File: src/App.vue (hunk 1 above)
Comment (francoMG) [reply src/App.vue:32]: Good improvement, did not occur to me 🙌

File: src/services/UsersService.js (hunk 3 above)
Comment (francoMG) [reply src/services/UsersService.js:5]: It's true, I always forget it 🤦‍♂️

**Comment by francoMG** [comment]
LG2M!

Thank you for the suggestions! See you soon 🤙

**Pull request merged**